### ISSUE TITLE
Update Digital Hour Cloud presentation based on AAACT recommendations

### ISSUE THREAD

**Issue by patheard** [dtf-ein/reading-material#32]
Notes from Jules:

* Remove alt text from decorative images.  Should be marked as decorative, or use alt text "Decorative" if using Office 2016.
* Increase minimum font size to 24 (related #31).
